ShareViaHttp/build.gradle

```
@@ -10,6 +14,14 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
+        mavenCentral()
+        maven { url "https://jitpack.io"}
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
```

1. The google maven repository should be before jitpack. Otherwise Gradle scans all dependencies on jitpack first. Which shouldn't be the case. 
2. Try to allow only those modules which should be picked by jitpack. This is a security feature
```suggestion
        google()
        maven { 
            url "https://jitpack.io"}
           content {
               includeGroup("com.github.kenglxn.QRGen")
            }
        }
```

---

tried with includeGroup, but it did not work...